content/2021/2021-04/2021-04-26-pulsar-weekly.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "2021-04-26-pulsar-weekly"
+displayTitle: "Pulsar Community Weekly Update - 2021-04-19 ~ 2021-04-25"
+date: 2021-04-19 ~ 2021-04-25
+description: "This is the Pulsar community weekly update for 2021-04-19 ~ 2021-04-25, with updates on Pulsar client, broker, transactions, and so on."
+id: "2021-04-26-pulsar-weekly"
+---
+
+## [Pulsar Community Weekly Update] 2021-04-19 ~ 2021-04-25
+
+This is the Pulsar community weekly update for 2021-04-19 ~ 2021-04-25, with updates on Pulsar client, broker, transactions, and so on.
+
+### Pulsar Highlight
+
+- Add some basic supports for testing de-duplication feature.
+
+    by ([@klwilson227](https://github.com/klwilson227))
+
+- Add `--topic-domain=persistent` and `--topic-domain=non-persistent` options to the pulsar-admin commands to get persistent topics and non-persistent topics.
+
+    by ([@codelipenghui](https://github.com/codelipenghui))
+
+- Upgrade Vert.x version to 3.9.7 to resolve vulnerability caused by version 3.5.3.
+
+    by ([@lhotari](https://github.com/lhotari))
+
+- Add basic configuration for the `org.owasp:dependency-check-maven` Maven plugin and make it operational in the Apache Pulsar project.
+
+    by ([@lhotari](https://github.com/lhotari))
+
+### Development
+
+- [PIP-82] [Broker] Add support for distributed resource quota enforcement at the tenant, namespace or topic level.
+
+    https://github.com/apache/pulsar/pull/10218 ([@ravi-vaidyanathan](https://github.com/ravi-vaidyanathan))
```

Comment:
This PR was merged 9 days before, shall we include it in this week?

Reply:
I’m not sure if we want to include it, it merged on 4.19 2:00 GMT+8

Reply:
if it's merged on 4/19, include it.